src/app/pages/research/components/members.js

```diff
@@ -0,0 +1,258 @@
+            <h2 className='pb-2' ref={membersRef}>{peopleHeader}</h2>
+            <TabAccordionCombo>
+                <div label='Current Members' selected>
+                    <MemberGrid>
+                        {members.map((member, index) =>
+                            <Member
+                                member={member}
+                                key={index}
+                                displayName={member.firstName}
+                            />
+                        )}
+                    </MemberGrid>
+                    <p
+                        className='py-4 mobile-only'
+                        onClick={() => {
+                            if (viewAll) {
+                                scrollToMembers();
+                            }
+                            setViewAll(!viewAll);
+                        }}
+                        css={{
+                            cursor: 'pointer',
+                            color: colors.linkText,
+                            textAlign: 'center'
+                        }}
+                    >
+                        <FontAwesomeIcon icon={viewAll ? faChevronUp : faChevronDown}></FontAwesomeIcon>
+                        &nbsp;
+                        {viewAll ? 'View Less' : 'View All Current Members'}
+                    </p>
+                </div>
+
+                <div label='Collaborating Researchers'>
+                    <CollaboratorGrid>
+                        {collaboratingResearchers.map((member, index) =>
+                            <Member
+                                member={member}
+                                key={index}
+                                displayName={`${member.firstName} ${member.lastName}`}
+                            />
+                        )}
+                    </CollaboratorGrid>
+                </div>
+
+                <div label='Alumni'>
+                    <AlumniGrid>
+                        {alumni.map((alumnus, index) =>
+                            <Alumnus alumnus={alumnus} key={index} />
+                        )}
+                    </AlumniGrid>
+                </div>
+            </TabAccordionCombo>
+        </Section>
+    );
+};
+
+const MemberModal = ({ member, isOpen, onHide }) => {
+    return (
+        <Dialog className='member-modal' onPutAway={onHide} isOpen={isOpen} closeOnOutsideClick={true}>
+            <MemberDetails member={member} />
+        </Dialog>
+    );
+};
+
+export const Member = ({ member, displayName }) => {
+    const [show, setShow] = useState(false);
+
+    return (
+        <Box direction='column' align='center' className='text-center'>
+            <img
+                height={145}
+                width={145}
+                alt={member.photo.title}
+                src={member.photo.file}
+                tabIndex={0}
+                onClick={() => {
+                    setShow(true);
+                    return false;
```

Comment:
You should probably be using `event.preventDefault()` rather than returning false.
https://deepscan.io/docs/rules/react-bad-event-handler-return-false/